meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Config.json watcher
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Service]
+Type=oneshot
+ExecStart=/bin/echo 'config.json changed'
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
Any reason to add this dependency to mnt-boot? We can use the default multi-user target because this looks like a dependency loop now.